wcs-functions.php

```diff
@@ -49,20 +49,24 @@ function wcs_is_subscription( $subscription ) {
 }
 
 /**
- * A very simple check. Basically if we have ANY subscriptions in the database, then the user has probably set at
- * least one up, so we can give them the standard message. Otherwise
+ * Return true if there are any subscriptions in the database (active or inactive).
  *
  * @since  1.0.0 - Migrated from WooCommerce Subscriptions v2.0
- * @return boolean true if anything is found
+ * @return boolean $subscriptions_exist True if the store has any subscriptions.
```

Comment:
```suggestion
 * @return bool True if the store has any subscriptions, otherwise false.
```

Per the WP documetnation standards `boolean` -> `bool` https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#phpdoc-tags